context/app/static/js/components/bulkDownload/bulkDownloadFormFields.ts

```diff
@@ -0,0 +1,16 @@
+import { z } from 'zod';
+import { withCustomMessage } from 'js/helpers/zod/withCustomMessage';
+
+const bulkDownloadOptionsField = {
+  bulkDownloadOptions: z
+    .array(z.string(), {
+      errorMap: withCustomMessage('My special error message'),
```

Comment:
This seems like it might be a placeholder?

Reply:
Sure is, good catch! :)